ChangeLog.d/fix_return_type_for_invalid_crypto_key.txt

```diff
@@ -0,0 +1,4 @@
+Bugfix
+   * Fixes the issue #4162. Returns PSA_ERROR_INVALID_HANDLE instead of
+     PSA_ERROR_DOES_NOT_EXIST for some key operations, if invalid key is
+     passed, to comply with PSA Crypto 1.0.0.
```

Comment:
This entry is missing some context to be comprehensible and has grammatical mistakes. Suggested text:

> PSA functions other than psa_open_key now return PSA_ERROR_INVALID_HANDLE rather than PSA_ERROR_DOES_NOT_EXIST for an invalid handle, bringing them in line with version 1.0.0 of the specification. Fix #4162.